_posts/2021-01-15-galaxy-wg.md

```diff
@@ -0,0 +1,24 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'New Galaxy Community Organization: Working Groups'
+tags: [galaxy]
+---
+
+<br>
```

Comment:
Is see this br more often what is the reason?

Reply:
the space between the header and the text does not seem enough to me :)

Reply:
I guess then we should tweak the CSS